rcl_interfaces/msg/IntegerRange.msg

```diff
@@ -1,15 +1,13 @@
 # Represents bounds and a step value for an integer typed parameter.
 
 # Lower bound for valid values, inclusive.
-int64 minimum_value
+int64 from_value
 
 # Upper bound for valid values, inclusive.
-int64 maximum_value
+int64 to_value
 
-# Size of valid steps between the lower and upper bound.
-# Step should be positive, and a step of zero implies a step size of one, i.e.
-# all integer values between the bounds are valid.
-# Step size should be less than the distance between the minimum and maximum
-# values, and ideally would be an even multiple of that distance, but that
-# is not required.
-int64 step
+# Size of valid steps between the from and to bound.
+# Step should be positive, and a step of zero implies continuous values.
+# Step size should be less than the distance between the values, and ideally
+# would be an even multiple of that distance, but that is not required.
```

Comment:
I think the suggested docs above apply here as well, but obviously need to be changed in a few ways for integers.